src/DependencyInjection/CompilerPass/SetDoctrineAnnotatedPrefixesPass.php

```diff
@@ -59,10 +59,9 @@ public function process(ContainerBuilder $container)
                     $methodCalls[$i] = $arguments;
                 }
 
-                $isAnnotated = false !== strpos($arguments[0], '_annotation_metadata_driver');
                 $annotatedPrefixes[$managerName][] = [
                     $arguments[1],
-                    $isAnnotated ? new Reference($arguments[0]) : null,
+                    new Reference($arguments[0]),
```

Comment:
Can you explain this change? thx :)

Reply:
The `DoctrineHelper::annoatedPrefixes` property stores all configured Namespaces for Doctrine with its drivers.
In `DoctrineHelper::getMappingDriverForNamespace` we check which Namespace (Prefix) uses which driver. For that we need to configure the `DoctrineHelper` in the pass with all configured drivers from the doctrine configuration and not only when they are using the Annotation driver.